packages/@aws-cdk/aws-cloudwatch/lib/graph.ts

```diff
@@ -181,6 +181,7 @@ export class GraphWidget extends ConcreteWidget {
         view: 'timeSeries',
         title: this.props.title,
         region: this.props.region || cdk.Aws.REGION,
+        stacked: this.props.stacked || false,
```

Comment:
To preserve current behavior, we should not emit `stacked` if it's `false` since this is the default, so this should be sufficient I believe, correct?

```ts
stacked: this.props.stacked
```

Reply:
That's correct, changed it.